src/index.js

```diff
@@ -15,6 +16,45 @@ const KeyCommand = NativeModules.KeyCommand ? NativeModules.KeyCommand : new Pro
     },
 );
 
+function validateKeyCommand(keyCommand) {
+    /**
+     * command object may have following schema:
+     * - {input: 'f', modifierFlags: 123}
+     * - {input: 'f'}
+     * - {input: '123'}
+     */
+    if (!keyCommand.input) {
+        // eslint-disable-next-line
+        console.error('input property for keyCommand object must be an provided. skipping.');
```

Comment:
if input is required, then it should throw a real runtime error. Otherwise there is no use of adding key listener if we are skipping. 
```suggestion
        
        throw new Error('Input property for keyCommand object must be provided.');
```

Reply:
updated